qutip/mesolve.py

```diff
@@ -127,8 +127,10 @@ def mesolve(H, rho0, tlist, c_ops=None, e_ops=None, args=None, options=None,
         single collapse operator, or list of collapse operators, or a list
         of Liouvillian superoperators.
 
-    e_ops : None / list of :class:`qutip.Qobj` / callback function single
-        single operator or list of operators for which to evaluate
+    e_ops : None / list of :class:`qutip.Qobj` / list of callback functions
+    / callback function
+        single operator, list of operators or list of callable functions
+        for which to evaluate
         expectation values.
```

Comment:
Could you rework this: 
It give the impression we cannot mix `Qobj` and `callable` in the list. `e_ops : None / list / callback function` with the description telling what is accepted in the list and that they can be mixed would be better.
- "expectation values." should be on the previous line.

Reply:
Certainly. Just updated.